internal/generator/utils.go

```diff
@@ -125,3 +125,11 @@ func checkDefaultValue(fieldType string, tag string) error {
 
 	return nil
 }
+
+func normalizeName(fieldName string) string {
```

Comment:
This function is `normalizeTypeName` I suppose. And argument is typeName. Am I right?